.travis.yml

```diff
@@ -1,20 +1,35 @@
-services: docker
+dist: "trusty"
+sudo: required
+
+matrix:
+  include:
+    - language: python
```

Comment:
I think this line overrides the `language:ruby` which makes the Travis don't run in `ruby` project mode again.

I have several ideas for this:
- make a separate job for the coala check with `language: python`
- make this job as `language: python` directly, since I think we're currently not using many of the `ruby` development tools

Any idea @jayvdb ?

Reply:
https://gitter.im/coala/coala-bears?at=5b44c27a3c5abf52b64b54b8 will solve this problem

Reply:
gitmate went nuts on https://github.com/coala/projects/pull/593 , so we may need to wait a bit.